src/components/Home.js

```diff
@@ -1,6 +1,49 @@
-import React from 'react';
+import React, { Fragment, Component } from 'react';
+import { Redirect } from 'react-router-dom';
+import Button from '@material-ui/core/Button';
 import '../App.css';
 
-const Home = () => (<h1 className='home'>Home</h1>);
+class Home extends Component {
+
+	constructor(props) {
+		super(props);
+		this.state = {
+			redirect: false,
+		}
+	}
+
+	componentWillMount = () =>  {
+		if(localStorage.getItem('authorizedUser')) {
+			console.log('userOk');
+		} else {
+			this.setState({redirect: true});
+		}
+	}
+
+	logout = () => {
+		localStorage.removeItem('authorizedUser');
```

Comment:
You have service for all calls of the localStorage.

Reply:
done